rma/models/rma.py

```diff
@@ -317,6 +317,107 @@ def _domain_location_id(self):
         copy=False,
     )
 
```

Comment:
Please avoid these empty lines for better readability of all the fields being shown together.